components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1198 @@
+ = \frac{1}{h} \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \varphi(x, y, z, t)  \, d\tilde{z}.
+$$ (def-tracer-avg)
+
+The conservation equation now becomes:
+
+$$
+\frac{d}{dt} \int_{A} h \, \overline{\varphi}^{\tilde{z}} \, dA
++ \int_{\partial A} h \, \overline{\varphi\mathbf{u}}^{\tilde{z}} \cdot \mathbf{n} \, dl
++ \int_{A} [\varphi \omega_{tr}]_{\tilde{z} = \tilde{z}^{\text{top}}} \, dA
+- \int_{A} [\varphi \omega_{tr}]_{\tilde{z} = \tilde{z}^{\text{bot}}} \, dA = 0,
+$$ (tracer-Aint-ztildeavg)
+
+where $\omega_{\text{tr}}$ is the mass transport per unit area through the top and bottom surfaces. Making use of their (A.19) and (A.20), we arrive at:
+
+$$
+\frac{d}{dt} \overline{h \, \overline{\varphi}^{\tilde{z}}}^A
++ \frac{1}{\tilde{A}} \int_{\partial A} h \, \overline{\varphi\mathbf{u}}^{\tilde{z}} \cdot \mathbf{n} \, dl
++ \left. \overline{\varphi \omega_{tr}}^A \right|_{\tilde{z} = \tilde{z}^{\text{top}}}
+- \left. \overline{\varphi \omega_{tr}}^A \right|_{\tilde{z} = \tilde{z}^{\text{bot}}} = 0,
+$$ (tracer-Aavg-ztildeavg)
+
+which is their (A.21), except with $\phi = \rho g \varphi$ and $w \rightarrow \omega$. Taking the limit $\tilde{A} \rightarrow 0$, we get:
+
+$$
+\frac{\partial}{\partial t} h \, \overline{\varphi}^{\tilde{z}} + \nabla \cdot (h \, \overline{\varphi\mathbf{u}}^{\tilde{z}} ) + [\varphi \omega_{tr}]_{\tilde{z} = \tilde{z}^{\text{top}}} - [\varphi \omega_{tr}]_{\tilde{z} = \tilde{z}^{\text{bot}}} = 0.
+$$ (tracer-layered)
+
+We get mass conservation by substituting $\varphi \rightarrow 1$:
+
+$$
+\frac{\partial}{\partial t} h + \nabla \cdot (h \overline{\mathbf{u}}^{\tilde{z}} ) + \left.\omega_{tr}\right|_{\tilde{z} = \tilde{z}^{\text{top}}} - \left.\omega_{tr}\right|_{\tilde{z} = \tilde{z}^{\text{bot}}} = 0.
+$$ (mass-layered)
+
+This is nearly identical to their (A.25) but with the pressure-thickness and $\omega$.
+
+
+## 5. Horizontal Momentum Equation
+
+We now derive the horizontal momentum equation in our non-Boussinesq, hydrostatic framework, following the same finite-volume approach used for mass and tracer conservation. We work with a pseudo-height vertical coordinate $\tilde{z}$ as defined in [Pseudo-Height Coordinate Section](pseudo-height).
+
+We begin by considering conservation of horizontal momentum density $ \rho \mathbf{u} $ over a time-dependent control volume $ V(t) $, bounded horizontally by a fixed area $ A $, and vertically by surfaces $ \tilde{z}^{\text{top}}(x, y, t) $ and $ \tilde{z}^{\text{bot}}(x, y, t) $. The finite-volume balance reads:
+
+$$
+\frac{d}{dt} \int_{V(t)} \rho \mathbf{u} \, dV
++ \int_{\partial V(t)} \rho \mathbf{u} (\mathbf{v} - \mathbf{v}_r) \cdot \mathbf{n} \, dA
+= \mathbf{F}_\text{total}[V(t)],
+$$ (momentum-FV)
+
+with total forces given by:
+
+$$
+\mathbf{F}_\text{total}[V(t)] =
+- \int_{V(t)} \rho\, \mathbf{f} \times \mathbf{u} \, dV
+- \int_{V(t)} \rho\, \nabla \Phi \, dV
+- \int_{\partial V(t)} p \, \mathbf{n} \, dA
```

Comment:
A notation suggestion: could we use dS (as a partner to dV) rather than dA. To me that makes more sense as dA is often used for fixed areas.

Reply:
Yes, I like that better since this is not the same dA as below.